### ISSUE TITLE
Turn the verse reference / document / etc into buttons

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#1336]
I was using another Bible app and it had the version in short form in a button at the top of the screen. I liked it. It felt right to click the version name to change to another version. 

I have also wanted to be able to choose a different chapter in the same book. At present I usually just swipe left or right because having to choose the correct book in the main selector is for me a bit slow - i can never find it straight away although the horizontal ordering is helping.

The suggestion below address both of those issues.

1. Turn the verse reference into a three part 'button'. This would make it possible to jump straight to either the chapter or the verse selector. 
2. Or it is a single button that takes you to the current selector for dictionaries etc
3. Turn the version name in to a button that opens the relevant document selector (eg the list of Bible translations). This is redundant because the user can already use the Bible icon. But it is useful for when the document is *not* a Bible. And it allows easy access to the document selector (don't need to know about the long press)

![image](https://user-images.githubusercontent.com/13920678/126040562-c75bfaef-2775-41ab-8209-4129649c3213.png)

![image](https://user-images.githubusercontent.com/13920678/126040568-3d8595f8-dc0e-4090-a63b-2feef56453ce.png)

**Comment by tuomas2** [comment]
they will be very small buttons. Not sure about usefulness because of that, but maybe.

**Comment by agrogers** [comment]
Yeah, I was trying to get a feel for it after I posted this. I think they will be at least as big as our colored label buttons. Small but not useless... unless there is an enforced top and bottom margin on that header that we can't easily change.

**Comment by tebicap** [comment]
Hi, I recently discovered this app and love the flexibility,  I was searching for feature-requests like this one because there is not way currently to jump between chapters or verses quickly (as far I know). Maybe keeping the main button as it is the book selector can be improved to handle chapters and verses, I'll leave here some suggestions to consider the developers. Thank you.


Vertical Chapter/verses selector,
at first the screen shows the curren chapter and verse position, clicking a chapter the verse column sets to "1" (dialog keeps opened), clicking a verse close the screen and goes back to the bible screen:
![Image](https://github.com/user-attachments/assets/b96a4a53-e109-4f86-a69d-97e2c49a0397)

Horizontal chapter/verses selector:
<img width="1080" height="1920" alt="Image" src="https://github.com/user-attachments/assets/7c6252ae-897d-4957-8a26-0e984dd91047" />

Scrolling chapter/verses selector (scrolling over the numbers it sets a new chapter or verse, or they can be just buttons to press and display the chapter/verse buttons screen)
![Image](https://github.com/user-attachments/assets/fb26b402-bed9-4e6e-9e08-fe0981b20e48)
